README.md

```diff
@@ -86,18 +87,31 @@ For scripting purposes, we output certain exit codes for differing scenarios.
 |2   |Parser error; check input to command line.   |
 |3   |Missing ARM resource; used for existence check from `show` commands.   |
 
-#### More Samples and Snippets
-For more usage examples, take a look at our [GitHub samples repo](http://github.com/Azure/azure-cli-samples) or [https://docs.microsoft.com/cli/azure/overview](https://docs.microsoft.com/cli/azure/overview).
+### Common scenarios and use Azure CLI effectively
```

Comment:
The "and" makes this sentence a little confusing.  Are "common scenarios" and using "Azure CLI effectively two different things?  Consider "that".

###Common scenarios that use Azure CLI effectively

Reply:
I think the contents of this section focuses on both aspects: common scenarios AND instructions for users to use CLI effectively. Not all scenarios are using CLI effectively. Some are for issue workarounds or troubleshooting.